### ISSUE TITLE
Unable to Add Tag Siblings in 407

### ISSUE THREAD

**Issue by GoAwayNow** [hydrusnetwork/hydrus#433]
## Environment

Hydrus version: 407
Platform: Windows
OS specifics: Windows 10 Pro v1903 x64

## What happens

When attempting to apply tag siblings in version 407, an error stating "'list' object has no attribute 'difference'" occurs.

## Steps to reproduce

1. Open the tag siblings management dialogue
2. Pend a new tag sibling, either locally or to the PTR
3. Press "Apply".

## Desired outcome

Tag sibling is either applied locally or added to the upload queue, depending on the service.

## Error message / Log file / Screenshots

```
AttributeError
'list' object has no attribute 'difference'
  File "hydrus\client\gui\ClientGUITopLevelWindows.py", line 531, in EventDialogButtonApply
  File "hydrus\client\gui\ClientGUITopLevelWindows.py", line 438, in _TryEndModal
  File "hydrus\client\gui\ClientGUITopLevelWindowsPanels.py", line 170, in _DoClose
  File "hydrus\client\gui\ClientGUITags.py", line 3652, in CommitChanges
  File "hydrus\client\gui\ClientGUITags.py", line 4403, in GetContentUpdates

```

**Comment by Zweibach** [comment]
> Hydrus version: please also state if you are running Hydrus via executable or from source. If you don't know what that means, you are very likely running via executable. If you are running from source, please include details on how (e.g., installed dependencies via pip, installed the AUR version etc.)

I'm running the Windows installer version and have the same problem.

**Comment by GoAwayNow** [comment]
My mistake, forgot to add that. I'm running the executable, zipped version.

**Comment by Miraihi** [comment]
Yeah, same problem. Occurs when I try to sibling namespaced tags.

**Comment by hydrusnetwork** [comment]
Argh, I am sorry for this. I got stuck in the lurch a bit with siblings last week, when I could not complete the new siblings display system. I think I must have accidentally half-applied something to the dialog here and not put aside time to test all components properly.

I will clean this all up for the bigger 408 siblings release, so please hang in there for that. I'll also figure out some unit tests so this doesn't happen again. Thank you for this report.
